pkg/ui/process-bindata.sed

```diff
@@ -0,0 +1,21 @@
+# This script post-processes the output of go-bindata.
+
+# TODO(benesch): this script would be much shorter and could fold into the
+# Makefile if goimports could automatically add the pkg/ui import, but goimports
+# currently can't handle files as large as bindata.go.
+
+# Add comment recognized by Reviewable to the top of the file.
+1i\
+// GENERATED FILE DO NOT EDIT
+
+# Add pkg/ui import statement in import block.
+/^import (/a\
```

Comment:
Could we just put this content in both `distccl/distccl.go` and `distoss/distoss.go`?

Reply:
Man, I wish. It would sure be cleaner, wouldn't it? Unfortunately because of the stupid way `go test pkg/...` works, it's essential that `pkg/ui/distccl` and `pkg/ui/distoss` are compilable whether or not the UI has been generated.

I'm somewhat comforted by the fact that one day the bug in goimports will be fixed, and the Makefile can just say:

```shell
echo 'func init() { ui.Asset = Asset; ui.AssetDir = AssetDir... }' > $@
goimports -s -w $@
```